Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -347,30 +344,16 @@ export class dotNetExe {
             } 
             return resolvedProjectFiles;
         }
-
         return projectFiles;
     }
 
     private isWebSdkUsed(projectfile: string) {
         var fileBuffer: Buffer = fs.readFileSync(projectfile);
-        var fileEncodeType = fileEncoding.detectFileEncoding(projectfile, fileBuffer);
-        var webConfigContent: string = fileBuffer.toString(fileEncodeType[0]);
-
-        if(fileEncodeType[1]) {
-            webConfigContent = webConfigContent.slice(1);
-        }
-
-        var ltxDom = new ltxdomutility.LtxDomUtility(webConfigContent);
-        var projectDomArray: Array<any> = ltxDom.getElementsByTagName("project");
-
-        for(var project of projectDomArray) {
-            if(project.attrs['Sdk'].toLowerCase() == "microsoft.net.sdk.web" || 
-                project.attrs['sdk'].toLowerCase() == "microsoft.net.sdk.web") {
-                return true;
-            }
-        }
+        var webConfigContent: string = fileBuffer.toString();
```

Comment:
fileEncoding might be interesting for reading the contents. May be you should copy the relevant parts of the code here

Reply:
Actually I tried with couple of file encodings, with BOM and without, and it seem to figure out the sdk without passing in the encoding explicitly.. also we'd need to write our own function to determine the encoding if we are to not import the webdeployment-common package..

Reply:
Let me know if this needs to be done, I'm waiting on your suggestions.